src/app/helptext/storage/disks/disk-form.ts

```diff
@@ -37,11 +36,21 @@ disk_form_smartoptions_tooltip : T('Additional <a\
  href="https://www.smartmontools.org/browser/trunk/smartmontools/smartctl.8.in"\
  target="_blank">smartctl(8)</a> options.'),
 
+disk_form_critical_placeholder: T('Critical'),
+disk_form_critical_tooltip: T('Report as critical in the system log and send an\
+ email if the temperature is greater or equal than N degrees Celsius. 0 to disable.'),
+
+disk_form_difference_placeholder: T('Difference'),
+disk_form_difference_tooltip: T('Report if the temperature has changed by at least N\
+ degrees Celsius since the last report. 0 to disable.'),
```

Comment:
Also existing help text for Difference field in Services > SMART. Suggested rework:
```
Enter a number of degrees Celsius. <i>0</i> disables this check. SMART reports if the temperature of a drive has changed by N degrees Celsius since the last report.
```

Reply:
Rearrange to tell what it is first: "Report if the temperature of a drive has changed by this many degrees Celsius since the last report. Entering <i>0</i> disables the report."